test/counterdb.test.js

```diff
@@ -127,9 +127,12 @@ Object.keys(testAPIs).forEach(API => {
 
         return new Promise(resolve => {
           // Wait for a while to make sure db's have been synced
-          setTimeout(() => {
+          setTimeout(async () => {
```

Comment:
could awaiting replicated events be good here?

Reply:
Those particular setTimeouts can actually go, since I moved that shim to [`orbit-db-test-utils`](https://github.com/orbitdb/orbit-db-test-utils/blob/master/stop-ipfs.js#L13)

As far as awaiting the replicated events, that could possibly help...

Reply:
if the replicated eventListeners arent needed they should be left out.

Reply:
Sorry, where are you seeing the replicated event listeners?

Reply:
Oh sorry. I mean if they aren't needed to make sure that both counters are utd, they don't need to be added.